tests/ebuild/test_atom.py

```diff
@@ -548,3 +548,23 @@ def test_unversioned(self):
         self.assertFalse(self.kls("=dev-util/diffball-1").is_simple)
         self.assertFalse(self.kls("dev-util/diffball[x]").is_simple)
         self.assertFalse(self.kls("dev-util/diffball[x?]").is_simple)
+
+    def test_get_atom_without_use_deps(self):
+        for original, wanted in (
```

Comment:
`pytest.mark.parametrize` perhaps?

Reply:
For now problematic, as I can't use fixtures as this test class inherits from `TestCase`.

In a separate branch I'm refactoring those test classes to not use it, so let's use ugly `for` for this PR?

Reply:
Sure. I thought this was post-refactoring already ;-).